docs/ui-components.md

```
@@ -0,0 +1,12 @@
+# UI Components
+
+`@promethean/ui-components` provides design tokens and reusable Web Components.
+
+## Design Tokens
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)